src/main/java/seedu/address/logic/parser/TaskHubParser.java

```diff
@@ -96,6 +98,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ListProjectCommand.COMMAND_WORD:
             return new ListProjectCommand();
 
+        case AddTaskCommand.COMMAND_WORD:
```

Comment:
I feel that this would be one of the more important segments of the code to test (you can just follow the style of the existing parser tests)